firmware/Makefile

```diff
@@ -2,6 +2,7 @@
 BOOTLOADER_SIZE = 0x1400
 LDFLAGS_FLASH = --code-loc $(BOOTLOADER_SIZE)
 USB_DEVICE_SERIAL_NUMBER="`./new_serial.py`"
+DEVICE_SERIAL_NUMBER="`cat .serial`"
```

Comment:
why the additional tracking of the serial#?  one serial number per device is preferred.  it should be the same one for usb as it is for the firmware check.